test/decl/var/property_wrappers.swift

```diff
@@ -1644,3 +1644,19 @@ struct UseNonMutatingProjectedValueSet {
     x = 42  // expected-error{{cannot assign to property: 'self' is immutable}}
   }
 }
+
+// SR-11478
+
+@propertyWrapper
+struct SR_11478_W<Value> {
+  var wrappedValue: Value
+}
+
+class SR_11478_C1 {
+  @SR_11478_W static var bool1: Bool = true // Ok
+  @SR_11478_W class var bool2: Bool = true // expected-error {{class stored properties not supported in classes; did you mean 'static'?}}
```

Comment:
Can you add a test case for `class final var` too? Since that fails today for normal stored properties it should probably continue to fail here, but we should have a test.

Reply:
Done